framer/Layer.coffee

```diff
@@ -81,6 +86,7 @@ class exports.Layer extends BaseClass
 		# Make sure we never call the constructor twice
 		throw Error("Layer.constructor #{@toInspect()} called twice") if @__constructed
 		@__constructed = true
+		@__constructing = true
```

Comment:
It's weird to set `@__constructing` after `@__constructed`, right? Maybe rename `@__constructed` to `@__constructorCalled` or something?